daemon/k8s_watcher.go

```diff
@@ -1630,5 +1659,45 @@ func (d *Daemon) deleteK8sPodV1(pod *v1.Pod) {
 	default:
 		logger.Debug("Deleted ipcache map entry on pod delete")
 	}
+}
+
+func (d *Daemon) updateK8sV1Namespace(oldNS, newNS *v1.Namespace) {
+	if oldNS == nil || newNS == nil {
+		return
+	}
 
+	// We only care about label updates
+	oldPodLabels := oldNS.GetLabels()
```

Comment:
variable naming is really bad here ;-) Just pass `xx.GetLabel()` into the `MapStringEquals()` to avoid this.

Reply:
Done